pyanaconda/storage_utils.py

```diff
@@ -47,7 +47,9 @@
 from pykickstart.constants import AUTOPART_TYPE_LVM, AUTOPART_TYPE_LVM_THINP
 
 import logging
```

Comment:
Should be done now - the anaconda_loggers module now has getters for the module loggers and for all the specialized loggers.